pyEpiabm/pyEpiabm/household_sweep.py

```diff
@@ -0,0 +1,56 @@
+#
+# Infection due to contact within households
+#
+
+from .abstract_sweep import AbstractSweep
+from .population import Population
+from .covidsim_helpers import CovidsimHelpers
+from .parameters import Parameters
+import random
+
+
+class HouseholdSweep(AbstractSweep):
+    '''Class to run the intra-household infections
+    as part of the sweep function. Takes an individual
+    person as input and tests a infection event against each
+    susceptible member of their household. The resulting
+    exposed person is added to an infection queue.
+
+    : param infected: Person
+    : return
```

Comment:
Hmm not sure what the convention is when nothing is returned, just an attribute of a different object changed?

Reply:
@rcw5890 I wanted to ask you about the conventions here anyway, because we've got a few going on in different parts of the code, and it's good to be consistent (and compatible with the docs)